assets/src/__tests__/util/math.test.js

```diff
@@ -34,23 +33,27 @@ describe('median', () => {
   })
 })
 
-describe('roundToOneDecimal', () => {
-  it('takes a rational number and returns that number rounded to one decimal place', () => {
-    expect(roundToOneDecimal(1)).toBe(1)
-    expect(roundToOneDecimal(1.5)).toBe(1.5)
-    expect(roundToOneDecimal(1.55)).toBe(1.6)
-    expect(roundToOneDecimal(1.555)).toBe(1.6)
-    expect(roundToOneDecimal(1.32)).toBe(1.3)
-  })
-})
-
-describe('roundToTwoDecimal', () => {
-  it('takes a rational number and returns that number rounded to two decimal place', () => {
-    expect(roundToTwoDecimal(1)).toBe(1)
-    expect(roundToTwoDecimal(1.5)).toBe(1.5)
-    expect(roundToTwoDecimal(1.55)).toBe(1.55)
-    expect(roundToTwoDecimal(1.551)).toBe(1.55)
-    expect(roundToTwoDecimal(1.555)).toBe(1.56)
-    expect(roundToTwoDecimal(1.32)).toBe(1.32)
+describe('roundToXDecimals', () => {
```

Comment:
Thanks for adding tests - I'm planning to add some more and look into how to set up automated E2E testing when there's a bit more time